admin/commands/pmm/client/upgrade.go

```diff
@@ -163,17 +163,17 @@ func (c *UpgradeCommand) getUpgradeCommands() ([][]string, error) {
 	switch cmd {
 	case common.Dnf:
 		return [][]string{
-			{"dnf", "upgrade", "-y", fmt.Sprintf("pmm2-client%s", c.getVersionSuffix(dnf))},
+			{"dnf", "upgrade", "-y", fmt.Sprintf("pmm3-client%s", c.getVersionSuffix(dnf))},
```

Comment:
the same about package names.

Reply:
@BupycHuk So, do we need to use pmm-client and pmm-server as the  package names? If this is the case, don't we have to change the package names from pmm2-client/server to pmm-client/server for PMM 2 as well?

Reply:
We don't need it for PMM 2, because it's a breaking change and will break a lot of stuff, but for PMM 3 we are doing breaking changes and we are ok with that.